pyproject.toml

```diff
@@ -30,17 +30,18 @@ copier = "copier.__main__:copier_app_run"
 python = ">=3.9"
 colorama = ">=0.4.6"
 dunamai = ">=1.7.0"
+eval-type-backport = { version = ">=0.1.3,<0.3.0", python = "<3.10" }
 funcy = ">=1.17"
 jinja2 = ">=3.1.4"
 jinja2-ansible-filters = ">=1.3.1"
+lazystuff = ">=1.2.0"
```

Comment:
We're likely introducing license poisoning again: https://github.com/ookisan/lazystuff/blob/main/LICENSE :see_no_evil:

I haven't tried it out, but I wonder whether a custom lazy dict implementation like https://stackoverflow.com/a/69948140 might work and be sufficient.